src/utils/data/application-data.js

```diff
@@ -0,0 +1,22 @@
+const convertData = (data) => {
+  if (data.organizationId) {
+    return {
+      id: data.applicationId,
+      orgId: data.organizationId._id,
+      ...data,
+    };
+  }
+  return {
+    id: data.applicationId,
+    ...data,
+  };
+};
+
+export default (data) => {
+  if (data.length && data[0].applicationId && data.constructor === Array) {
+    return data.map((item) => convertData(item));
+  } else if (data.applicationId && data === Object(data)) {
```

Comment:
By virtue of the implications stated in the previous review comment, you can omit the second condition as the above if verifies that it isn't an object.

This means that it can either be an object or some literal value (e.g. string or number).

Which leads me to this, if `data.application` is truthy, then it must be an object. So the second condition `data === Object(data)` is not really necessary.